src/register.coffee

```diff
@@ -104,6 +104,9 @@ module.exports = (options={}) ->
 
     replaceHandler = (extension) ->
         origCoffeeHandler = require.extensions[extension]
+        # just expose needed API in some hideous way....
+        fs.instrumentFile ||= (fileName) ->
```

Comment:
Why are we adding this to `fs`?  O_o  Seems like adding this to the main coffeeCoverage export would be a better plan?